Updated the "Configure autoscale rules" PowerShell script for Simplicity

pranoybej opened the pull request:
Updated the "Configure autoscale rules" PowerShell Script for Simplicity.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@jimdial - Could you please review this public repo PR?

jimdial commented:
#sign-off

4c74356b41 commented:
Hello?

pranoybej committed:
Updated the "Configure autoscale rules" PowerShell script for Simplicity

pranoybej committed:
Update tutorial-create-vmss.md

I have removed the unnecessary content from line no 225-272 as per your suggestion.
Thanks for pointing out.

pranoybej committed:
Update tutorial-create-vmss.md

pranoybej committed:
Update tutorial-create-vmss.md

pranoybej committed:
Added the Missing div class on line 225.

Thank you so much for pointing out.
I hope now you can merge this.
